### DISCUSSION TITLE
Unable to Log In with Password on Terminal of Codespace for Docker

### DISCUSSION THREAD

**Discussion by LenaKemp** [ua-gist604b-s24/syllabus#21] [Q&A] [answered]
Inside the Terminal Panel to complete "Share Application" section of the Docker tutorial, I am unable to type anything in for my Password. However, I am logged in on both the Docker Desktop application and the Docker website. I watched the professor's zoom recording, but we do not actually see what he did off screen to get it to work. Please view attachment.
<img width="414" alt="Username_Password_Error" src="https://github.com/ua-gist604b-s24/syllabus/assets/163209046/bb7f1c01-5aa2-4240-81ff-949d9a10986b">

Kind regards

**Comment by aaryno** [comment, marked as answer]
When you are in a terminal window, password prompts typically turn off the "echo" of your keyboard. You can type in your password but it will not be shown on the screen in case some prying eyes are looking over your shoulder and steal your password. 

In your example above, you can simply type your password (even though you can't see it) and press <ENTER>.

**Reply by Ninad-Golatkar** [reply]
I was stuck here for quite some time, thank you!
